src/openrct2/windows/map.c

```diff
@@ -1349,41 +1347,32 @@ static void window_map_place_park_entrance_tool_down(sint32 x, sint32 y)
  */
 static void window_map_set_peep_spawn_tool_down(sint32 x, sint32 y)
 {
-    rct_map_element *mapElement, *surfaceMapElement;
+    rct_map_element *mapElement;
     sint32 mapX, mapY, mapZ, direction;
 
     footpath_get_coordinates_from_pos(x, y, &mapX, &mapY, &direction, &mapElement);
-    if (mapX == 0x8000)
-        return;
-
-    surfaceMapElement = map_get_surface_element_at(mapX >> 5, mapY >> 5);
-    if (surfaceMapElement == NULL) {
-        return;
-    }
-    if (surfaceMapElement->properties.surface.ownership & 0xF0) {
-        return;
-    }
-
-    mapX = mapX + 16 + (word_981D6C[direction].x * 15);
-    mapY = mapY + 16 + (word_981D6C[direction].y * 15);
```

Comment:
This is setting the map position to be one unit in front of the edge (of `direction`). Leave it be.